Add Persian translation

Mohammaddar opened the pull request.

Mohammaddar commented:
@googlebot I signed it!

LinX64 commented:
I already added a pull request for FA translation here: https://github.com/android/sunflower/pull/642

But they do not seem to have willing to accept it.

Mohammaddar commented:
> I already added a pull request for FA translation here: #642
> 
> But they do not seem to have willing to accept it.

Yeah. I saw your work after I added a pull request
